circle.yml

```diff
@@ -32,6 +32,7 @@ test:
   override:
     - ./scripts/circle-ci/run-circle-tests.sh:
         parallel: true
+        timeout: 900
```

Comment:
Do we actually need this timeout? What was it before? Feels like we can make our tests run faster, possibly mocking a Clock